### PR TITLE
[DOCFIX] How to use Alluxio for external hive table

### PR DESCRIPTION
**Pull request by apc999**

### CODE REVIEW

File: docs/en/Running-Hive-with-Alluxio.md [hunk 1]
```diff
@@ -21,10 +21,53 @@ set up in accordance to these guides for either [Local Mode](Running-Alluxio-Loc
...
+zipcode STRING)
+OW FORMAT DELIMITED
+FIELDS TERMINATED BY '|'
+LOCATION 'alluxio://localhost:19998/ml-100k';
+```
+
+## 2 Use Alluxio as Hive Default Filesystem
+
+Apache Hive also allows you to use Alluxio through a generic file system wrapper to replace the
+Hadoop file system. In this way, the Hive uses Alluxio as the default file system and Hive’s Hadoop jobs will store its intermediate results in Alluxio.
```
Comment (calvinjia) [inline docs/en/Running-Hive-with-Alluxio.md]: 100 characters

File: docs/en/Running-Hive-with-Alluxio.md [hunk 2]
```diff
@@ -21,10 +21,53 @@ set up in accordance to these guides for either [Local Mode](Running-Alluxio-Loc
...
 Please [Download Hive](http://hive.apache.org/downloads.html).
 
-## Configuration
+To run Hive on Hadoop MapReduce, please also follow the instructions in
+[running MapReduce on Alluxio](Running-Hadoop-MapReduce-on-Alluxio.html) to make sure Hadoop
+MapReduce can run with Alluxio.
 
-Apache Hive allows you to use Alluxio through a generic file system wrapper for the Hadoop file system.
-Therefore, the configuration of Alluxio is done mostly in Hive and its under computing frameworks.
+## 1 Use Alluxio as External Table Storage
```
Comment (calvinjia) [inline docs/en/Running-Hive-with-Alluxio.md]: Do we need numbering?

File: docs/en/Running-Hive-with-Alluxio.md [hunk 3]
```diff
@@ -21,10 +21,53 @@ set up in accordance to these guides for either [Local Mode](Running-Alluxio-Loc
...
+
+Hive can create external tables stored on Alluxio. The setup is easy and requires no change to
+existing Hive tables..
```
Comment (calvinjia) [inline docs/en/Running-Hive-with-Alluxio.md]: Extra `.`? Also maybe add a user case for why someone may want to store external tables in Alluxio?

File: docs/en/Running-Hive-with-Alluxio.md (hunk 2 above)
Comment (apc999) [inline docs/en/Running-Hive-with-Alluxio.md]: removed

File: docs/en/Running-Hive-with-Alluxio.md (hunk 3 above)
Comment (apc999) [inline docs/en/Running-Hive-with-Alluxio.md]: removed

File: docs/en/Running-Hive-with-Alluxio.md (hunk 1 above)
Comment (apc999) [inline docs/en/Running-Hive-with-Alluxio.md]: fixed

**Comment by apc999** [comment]
@calvinjia comments addressed

File: docs/en/Running-Hive-with-Alluxio.md [hunk 4]
```diff
@@ -21,10 +21,55 @@ set up in accordance to these guides for either [Local Mode](Running-Alluxio-Loc
...
 Please [Download Hive](http://hive.apache.org/downloads.html).
 
-## Configuration
+To run Hive on Hadoop MapReduce, please also follow the instructions in
+[running MapReduce on Alluxio](Running-Hadoop-MapReduce-on-Alluxio.html) to make sure Hadoop
+MapReduce can run with Alluxio.
 
-Apache Hive allows you to use Alluxio through a generic file system wrapper for the Hadoop file system.
-Therefore, the configuration of Alluxio is done mostly in Hive and its under computing frameworks.
+## Use Alluxio as External Table Storage
```
Comment (gpang) [inline docs/en/Running-Hive-with-Alluxio.md]: In the intro, you should specify that there are 2 ways to run hive on Alluxio. Also, is one method recommended?

Comment (apc999) [inline docs/en/Running-Hive-with-Alluxio.md]: @gpang added

**Comment by apc999** [comment]
@gpang @calvinjia PTAL

**Pull request merged**